docker-compose.yml

```diff
@@ -74,6 +74,7 @@ services:
       - ./apps/api-server:/opt/openstad-headless/apps/api-server
       - openstad-api-server-node_modules:/opt/openstad-headless/apps/api-server/node_modules
       - openstad-api-server-init:/opt/openstad-headless/apps/api-server/init
+      - ./packages:/opt/openstad-headless/packages
```

Comment:
Kan het toevoegen van de packages aan de docker mogelijk bijwerkingen hebben op de algehele docker-container? Bijvoorbeeld dat een WIP widget die nog niet juist opgezet is ervoor kan zorgen dat de container een error geeft. Of is daar niet echt een risico?